.github/workflows/windows.yml

```diff
@@ -0,0 +1,131 @@
+name: Windows
+
+on:
+  push:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  windows:
+    name: Windows
+
+    defaults:
+      run:
+        shell: pwsh
+
+    runs-on: windows-2022
+
+    env:
+      COMPOSER_NO_INTERACTION: '1'
+      SYMFONY_DEPRECATIONS_HELPER: 'strict'
+      ANSICON: '121x90 (121x90)'
+      SYMFONY_PHPUNIT_DISABLE_RESULT_CACHE: '1'
+
+    steps:
+      - name: Setup Git
+        run: |
+          git config --global core.autocrlf false
+          git config --global user.email ""
+          git config --global user.name "Symfony"
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Setup PHP
+        run: |
+          $env:Path = 'c:\php;' + $env:Path
+          mkdir c:\php && cd c:\php
+          iwr -outf php-7.2.5-Win32-VC15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php-7.2.5-Win32-VC15-x86.zip
+          7z x php-7.2.5-Win32-VC15-x86.zip -y >nul
+          cd ext
+          iwr -outf php_apcu-5.1.19-7.2-ts-vc15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php_apcu-5.1.19-7.2-ts-vc15-x86.zip
```

Comment:
how do failure on such command behave ? Do they fail the step or no ?

Reply:
/cc @wouterj

Reply:
you can try that easily: make a change in the URL so that the download fails, and see what happens in the run.

Reply:
(Disclaimer: I know Powershell for 2 days now)

By default, it doesn't stop on error for cmdlets. I don't know if this applies to exe processes as well.

But for iwr, which is a cmdlet, it means it will not fail. We can add `$ErrorActionPreference = 'Stop'` to stop on error for cmdlets.

Reply:
it looks like we don't need to add that: https://github.com/symfony/symfony/actions/runs/11610577132/job/32330130384?pr=58721#step:4:56